static/src/js/components/KeywordRecommendations/KeywordRecommendations.jsx

```diff
@@ -0,0 +1,344 @@
+        {' '}
+        for your collection. To associate a recommended keyword to your collection, click the
+        {' '}
+        <i className="keyword-recommendations__add-icon fa fa-plus-circle" />
+        {' '}
+        icon next to the keyword. Once associated to the collection, the keyword will display a
+        green check
+        {' '}
+        <i className="keyword-recommendations__accept-icon fa fa-check-square" />
+        . To remove a keyword once it’s been associated, click the
+        {' '}
+        <i className="keyword-recommendations__remove-icon fa fa-times-circle" />
+        {' '}
+        icon next to the keyword.
+      </div>
+    </div>
+  )
+
+  /**
+ * Creates a delimited keyword from the keyword's metadata
+ * @param {*} keywordObject
+ * @returns the keyword delimited with '>'
+ */
+  const createDelimitedKeyword = (keyword) => {
+    const delimitedKeyword = Object.values(keyword).join(' > ')
+
+    return delimitedKeyword
+  }
+
+  /**
+   * Create an array from a delimited keyword
+   * @param (*) keyword the delimited keyword
+   * @returns the keyword as an array
+   */
+  const createArraySeparatedKeyword = (keyword) => keyword.split('>').map((value) => (value.trim()))
+
+  /**
+   * Adds a keyword to the formdata
+   * @param {*} keyword the > delimited keyword
+   */
+  const addKeyword = (keyword) => {
+    const fields = [
+      'Category',
+      'Topic',
+      'Term',
+      'VariableLevel1',
+      'VariableLevel2',
+      'VariableLevel3']
+
+    const map = {}
+    fields.forEach((field) => {
+      map[field] = null
+    })
+
+    const arrayKeyword = createArraySeparatedKeyword(keyword)
+    arrayKeyword.forEach((item, index) => {
+      map[fields[index]] = item
+    })
+
+    formData.push(map)
+    onChange(formData)
+  }
+
+  /**
+   * Removes a keyword from form data
+   * @param {*} keyword the > delimited keyword
+   */
+  const removeKeyword = (keyword) => {
+    const existingKeywords = removeEmpty(cloneDeep(formData))
+    const updatedKeywords = existingKeywords.filter((draftKeyword) => {
+      const keyword1 = Object.values(draftKeyword).join(' > ')
+
+      return keyword !== keyword1
+    })
+
+    onChange(updatedKeywords)
+  }
+
+  /**
+   * Retrieves out of form data a list of Keyword objects
+   * of the form { keyword: string, recommended: bool, accepted: bool }
+   * @returns list of keyword objects
+   */
+  const getDraftKeywords = () => {
+    const keywordList = removeEmpty(cloneDeep(scienceKeywords))
+    const keywords = keywordList.map((keyword) => {
+      const delimitedKeyword = createDelimitedKeyword(keyword)
+
+      const draftKeyword = {
+        keyword: delimitedKeyword,
+        recommended: false,
+        accepted: true
+      }
+
+      return draftKeyword
+    })
+
+    return keywords
+  }
+
+  /**
+   * Returns true if the delimited Keyword is in the recommended keyword list.
+   * @param {*} keywordRecommendations the array of recommendations
```

Comment:
Is this `*` a wild card? We should be defining these types as best we can. Looks like there are a lot of these.